fix-webm-duration.js

```diff
@@ -1,4 +1,12 @@
-(function() {
+(function (name, definition){
+    if (typeof define === 'function'){ // AMD
```

Comment:
"define" function may exist. Please add a check for define.amd

Reply:
Added